src/components/Services.jsx

```diff
@@ -21,6 +23,26 @@ const Services = () => (
         <p className="text-left my-2 text-white font-light md:w-9/12 w-11/12 text-base">
           Offer you convenient, transparent and rapid services to donate safely around the world.
         </p>
+        <a
+          href={`https://ropsten.etherscan.io/address/${contractAddress}`}
+          target="_blank"
+          rel="noreferrer"
+          className="text-left my-2 text-white font-light md:w-9/12 w-11/12 text-base hover:text-blue-400"
+        >
+          <p>
+            Contract address: {shortenAddress(contractAddress)}
```

Comment:
```suggestion
            Contract address: <b className="text-cyan-300">{shortenAddress(contractAddress)}</b>
```

Suggest adding colors to the contract address for emphasis.

Reply:
resolve here [daface6](https://github.com/lohsuan/puppy_sponsor/pull/13/commits/daface6eb13f0406e6b1ac018a3b58ce09b2ae4f)